models/archive-exhibition.php

```diff
@@ -327,6 +366,73 @@ public function years() {
         return $choices;
     }
 
+    /**
+     * Reorder main exhibitions to top of other exhibitions with the same dates
+     *
+     * @param array $items Array of WP_Post instances.
+     *
+     * @return array
+     */
+    protected function reorder_main_exhibitions( $items ) {
+
+        // Return original $items array if search or year filter is used
+        if ( self::get_search_query_var() || self::get_year_query_var() ) {
+            return $items;
+        }
+
+        // Return null if there are no items in array
+        if ( $items === false ) {
+            return;
+        }
+
+        $items  = array_values( $items ); // reset array keys to start from 0
+        $length = count( $items );
```

Comment:
Tän voi poistaa ja tehdä
```php
foreach ( $items as $key => $value ) {
    unset( $items[ $key ] );
}
```

Reply:
Tekeeköhän unset saman, jota oon tuossa yrittäny? Tarkoituksena siis saada vaan arrayn keyt alkamaan nollasta ylöspäin

Reply:
Testasin tuolla foreachilla lokaalissa, se poistaa kaikki itemit listauksesta vaikka ne halutaan pitää olemassa